app.css

```diff
@@ -267,15 +283,129 @@ footer {
   align-items: center;
   text-align: center;
 }
-.contact-button:hover {
+.cont-button:hover {
   background-color: #4c73d6;
   color: white;
 }
-.contact-button:active {
+.cont-button:active {
   background-color: #4c73d6;
   color: white;
 }
-.contact-button:disabled {
+.cont-button:disabled {
   background-color: #6276a8;
   color: white;
 }
+
+@media screen and (max-width: 768px) {
```

Comment:
### OPTIONAL
- [ ] Kindly use min-width instead of max width, this helps you target screen sizes larger the 768px 🤝